### PR TITLE
Updated Challenge-01 README.md

### PR DESCRIPTION
**Pull request by SrikantManche**
Updated Challenge-01 README.md

### CODE REVIEW

**Comment by dhayanands** [comment]
@SrikantManche you can update the readme with below details:

- what is the playbook for?
- how to run the playbook

use markdown to update readme

**Comment by SrikantManche** [comment]
@dhayanands @zeusbaba Can you please review the checked-in code and provide comments

**Review by zeusbaba — APPROVED** [review]
well done @SrikantManche

File: challenge-01/Apache2WebServer.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+- name : Install Apache2 Web Server
+  hosts: localhost
+  become: yes
+  
+  vars:
```
Comment (phamp890) [inline challenge-01/Apache2WebServer.yml:6]: Instead of repeating _apache2_ you are using variable. Nice done

File: challenge-01/README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
-# title goes here 
-task description goes here...  
-TODO...  
+# Install Apache Server 
+
+## 1. Catalog.yml Playbook
```
Comment (phamp890) [inline challenge-01/README.md:3]: How do you run this playbook?

**Review by phamp890 — APPROVED** [review]
Well done @SrikantManche

**Comment by zeusbaba** [comment]
well done @SrikantManche

**Pull request merged**